frontend/app/components/activity-new.js

```diff
@@ -34,14 +34,31 @@ export default Component.extend(EKMixin, {
     context.sendAction("done", true);
   },
 
+  addMissingMonthWarning(newActivity) {
```

Comment:
I would divide the boolean return and the display.style stuff into separate methods... The returned boolean is a check on the record and the style stuff is specific to this component...

Refactor this to something like `showMissingMonthWarning()` and only do the style stuff here